packages/cli/src/commands/dev.ts

```diff
@@ -4,19 +4,33 @@ import { showWelcomeScreen } from './init.consts';
 import * as ora from 'ora';
 import * as open from 'open';
 import * as chalk from 'chalk';
+import { SERVER_HOST } from '../constants';
 
 process.on('SIGINT', function () {
-  console.log('Caught interrupt signal');
   // TODO: Close the NTFR Tunnel
   process.exit();
 });
 
+export enum WebRegionEnum {
+  US = 'us',
+  EU = 'eu',
+  STAGING = 'staging',
+}
+
+export enum WebUrlEnum {
+  US = 'https://web.novu.co',
+  EU = 'https://eu.web.novu.co',
+  STAGING = 'https://dev.web.novu.co',
+}
+
+const TUNNEL_URL = 'https://ntfr.dev/api/tunnels';
+
 export type DevCommandOptions = {
   port: string;
   origin: string;
-  region: 'us' | 'eu';
+  region: `${WebRegionEnum}`;
   studioPort: string;
-  studioRemoteOrigin: string;
+  webUrl: string;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ web => dashboard